traffic_ops/traffic_ops_golang/deliveryservice/capacity.go

```diff
@@ -135,6 +139,20 @@ func getCapacity(tx *sql.Tx, ds tc.DeliveryServiceName, cdn tc.CDNName) (Capacit
 	}, nil
 }
 
+// To check whether a given DS has servers.
```

Comment:
Nitpick: GoDoc comments need to be complete sentences starting with the symbol name. Should be
```
 // dsHasServer checks whether a given DS has servers. 
```

Reply:
Updated.